src/main/java/Eln.java

```diff
@@ -0,0 +1,47 @@
+
+import java.util.Scanner;
+
+
+public class Eln {
+    static String LINE = "____________________________________________________________";
```

Comment:
Besides that, I think that you made good note to use capital letters in "LINE" to use for this constant. Nice!